kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkAdminServiceImpl.java

```diff
@@ -242,43 +244,47 @@ protected List<WifiAccessPoint> getWifiAccessPoints(String ifaceName) throws Kur
     }
 
     @Override
-    // FIXME: This api should be deprecated in favor of the following signature:
-    // List<? extends NetInterfaceConfig<? extends NetInterfaceAddressConfig>> getNetworkInterfaceConfigs()
+    public List<? extends NetInterfaceConfig<? extends NetInterfaceAddressConfig>> getNetworkInterfaceConfigs(
+            boolean recompute) throws KuraException {
+
+        logger.debug("Getting all networkInterfaceConfigs");
+        long started = System.currentTimeMillis();
+        List<? extends NetInterfaceConfig<? extends NetInterfaceAddressConfig>> nc = getNetworkConfigurationInternal(
+                recompute).getNetInterfaceConfigs();
+        logger.info("GetNetworkConfiguration {}", (System.currentTimeMillis() - started));
```

Comment:
Probably better to look at the logging before merging since it logs massively now